client/components/Users/UserPaginator.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+export default class UserPaginator extends React.Component {
+  static propTypes = {
+    onPageChange: React.PropTypes.func.isRequired,
+    nextPage: React.PropTypes.number.isRequired,
+    pages: React.PropTypes.number.isRequired
+  }
+
+  shouldComponentUpdate(nextProps) {
+    return this.props.nextPage !== nextProps.nextPage;
+  }
+
+  render() {
+    const { onPageChange, nextPage, pages } = this.props;
+
+    return (
+      <nav className="pull-right">
+        <ul className="pagination pull-right">
+          <li
+            style={{ cursor: 'pointer' }}
+            onClick={() => onPageChange(nextPage - 1)}
```

Comment:
Don't create inline functions:

Create a new function in the class: `onPreviousPage = () => ...`